client/src/components/Proposals/ProposalsList/ProposalsList.tsx

```diff
@@ -93,7 +93,7 @@ const ProposalsList: FC<ProposalsListProps> = ({
               proposalIpfsWithRewards={proposalIpfsWithRewards}
             />
           ))
-        : (proposalsAddresses || []).map((_, index) => (
+        : proposalsAddresses?.map((_, index) => (
```

Comment:
This will not cause the client to crash -> even if `proposalsAddresses` will be undefined the optional chaining operator prevents the application from crashing.